### PR TITLE
Improve docs on PostgreSQL Extensions

### PR DESCRIPTION
**Pull request by lillo42**

### CODE REVIEW

File: src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs [hunk 1]
```diff
@@ -44,38 +43,5 @@ public static ICreateBTreeIndexOptionsSyntax Fillfactor(this ICreateIndexOptions
             return expression.UsingBTree()
                 .Fillfactor(fillfactor) as ICreateBTreeIndexOptionsSyntax;
         }
-
```
Comment (jzabroski) [inline src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs:47]: @lillo42 Why is this PR about improving the docs but you're deleting code?  Everything else looks good to me.  Just making sure you did this on purpose.

**Review by jzabroski — CHANGES_REQUESTED** [review]

**Review by fubar-coder — CHANGES_REQUESTED** [review]

File: src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs (hunk 1 above)
Comment (jzabroski) [reply src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs:47]: @lillo42  Any answer here?  Thanks.

File: src/FluentMigrator.Extensions.Postgres/Builder/Create/Index/IndexMethod/GistBuffering.cs [hunk 2]
```diff
@@ -16,10 +16,24 @@
...
+        /// </summary>
+        Auto,
+
+        /// <summary>
+        /// Enabled buffering
+        /// </summary>
         On,
-        Off,
-        Auto
+
```
[the remark this answers is not in the document]
Comment (jzabroski) [reply src/FluentMigrator.Extensions.Postgres/Builder/Create/Index/IndexMethod/GistBuffering.cs:33]: @fubar-coder I think in this case we should accept it.  On and Auto are very similar conceptually.  And I believe Postgres's default is Auto.  I should've caught that in the initial review but was not thinking about that as something with a default value.

[the remark this answers is not in the document]
Comment (fubar-coder) [reply src/FluentMigrator.Extensions.Postgres/Builder/Create/Index/IndexMethod/GistBuffering.cs:33]: TBH I don't think that it'll be a problem in practice, just a potential one.

File: src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs (hunk 1 above)
Comment (fubar-coder) [reply src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs:47]: I'm curious about this change too. However, I do like that the constants are moved to the `PostgresExtensions` file even though that it'll be a breaking change too (at source level).

**Review by fubar-coder — CHANGES_REQUESTED** [review]

File: src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs (hunk 1 above)
Comment (lillo42) [reply src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs:47]: Sorry for delay, I was moving last week, a lot of thing to do, about the delete code I realize that I have forgot to delete this code after move the storage parameter to the interfaces, maybe it's better rollback it and add Obsolete attribute, what do you think?

Comment (fubar-coder) [reply src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs:47]: If there is a replacement for those functions and it didn't exist for too long, then I agree that the breaking change might not be too disruptive for the FM users out there.

**Review by fubar-coder — APPROVED** [review]
I think that the risk of this breaking change is low, so I guess that it'll be fine to merge.

**Pull request merged**